apps/web/composables/useCategoryTemplate/homepageTemplateDataEn.json

```
@@ -203,5 +208,47 @@
         "label": "Subscribe"
       }
     }
+  },
+  {
+    "name": "Footer",
+    "type": "content",
+    "meta": {
+      "uuid": "4f7e2b1a-9c8d-4e6f-b3a2-1d0c5b4a3e2f",
+      "isGlobalTemplate": true
+    },
+    "content": {
+      "column1": {
+        "text": {
+          "title": "Legal"
+        }
+      },
+      "column2": {
+        "text": {
+          "title": "Contact",
+          "showLinkToContact": false,
+          "htmlDescription": "<ul><li><a href='https://plentyshop.plentymarkets.com/contact'></a></li></ul>"
```

```suggestion
          "htmlDescription": "<ul><li><a href='https://plentyshop.plentymarkets.com/en/contact'></a></li></ul>"
```
Same as above, can we not do `/en/contact`?